tests/Backend/AMQPBackendDispatcherTest.php

```diff
@@ -60,6 +62,48 @@ public function testThrowIfFactoryClassIsNotInstanceOfAmqpConnectionFactoryInter
         $dispatcher->getContext();
     }
 
+    public function testThrowIfDelayStrategyClassIsSetButNotRealClass()
```

Comment:
```suggestion
    public function testThrowIfDelayStrategyClassIsSetButNotRealClass(): void
```

Reply:
@greg0ire Should the return type be added to all the tests in the class or just the new ones?

Reply:
You can add it in a separate commit if you feel like helping :)